src/storage/v2/disk/storage.cpp

```diff
@@ -106,7 +106,9 @@ bool VertexHasLabel(const Vertex &vertex, LabelId label, Transaction *transactio
         break;
       }
       case Delta::Action::DELETE_DESERIALIZED_OBJECT:
-      case Delta::Action::DELETE_OBJECT:
+      case Delta::Action::DELETE_OBJECT: {
+        break;
+      }
```

Comment:
Same error at line 140

Reply:
can you explain?

Reply:
`break` was missing after `DELETE_OBJECT`
There is a similar switch at line 140 where the break is missing as well.

Reply:
was missing but now is okay or? Sorry I don't understand :smile: And about line 140, I think there is missing break yes I will add it now

Reply:
this change is correct :D yes